wicket-jquery-ui-parent/wicket-jquery-ui-calendar/src/main/java/com/googlecode/wicket/jquery/ui/calendar6/CalendarAdapter.java

```diff
@@ -0,0 +1,128 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.googlecode.wicket.jquery.ui.calendar6;
+
+import java.time.LocalDate;
+import java.time.LocalDateTime;
+
+import org.apache.wicket.ajax.AjaxRequestTarget;
+
+/**
+ * Adapter class for {@link ICalendarListener}
+ *
+ * @author Sebastien Briquet - sebfz1
+ */
+public class CalendarAdapter implements ICalendarListener
```

Comment:
Since we have default methods in interfaces these days in Java, such adapter classes should no longer be necessary.